src/components/LeftSidebar/ConversationsList/ConversationsListVirtual.vue

```diff
@@ -27,11 +27,11 @@ import LoadingPlaceholder from '../../UIShared/LoadingPlaceholder.vue'
 import 'vue-virtual-scroller/dist/vue-virtual-scroller.css'
 
 /* Consider:
- * 36 = 2 * 1.2 * var(--default-font-size) - two lines of text
+ * 39 = 2 * 1.3 * var(--default-font-size) - two lines of text
  * 8 = 2 * var(--default-grid-baseline) - item padding
  * 4 = var(--default-grid-baseline) - item outline (collapsed)
  */
-const CONVERSATION_ITEM_SIZE = 48
+const CONVERSATION_ITEM_SIZE = 51
```

Comment:
To not check that comment is true.
Or with variables.
```suggestion
const CONVERSATION_ITEM_SIZE = 2 * 1.3 * 15 + 2 * 4 + 4
```

Reply:
If we have a constant for avatar size, then it's SIZE + 2 * 4 + 4